src/bin/graph.rs

```diff
@@ -6,6 +6,9 @@
 //! (The resulting images are large, so the SVG format is recommended for
 //! quality-conserving zooming.)
 //! The 'dot' utility can be found in the 'graphviz' package.
+
+#![cfg_attr(feature="cargo-clippy", allow(doc_markdown))]
```

Comment:
Well, it's only for the `graph` binary, and the problem lines are those which show the recommended usage: e.g. `graph ewok_log_file -o output_file`.  Clippy wants `ewok_log_file` wrapped in backticks, but that wouldn't then be a correct usage example.